src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2375,7 +2375,10 @@ static void updateAttrInNotifyCer
 static void deleteAttrInNotifyCer
 (
   ContextElementResponse* notifyCerP,
-  ContextAttribute*       targetAttr
+  ContextAttribute*       targetAttr,
+  bool                    useDefaultType,
+  const std::string&      actionType,
+  const bool&             overrideMetadata
```

Comment:
It seems useDefaultType, actionType and overrideMetadata parameter are not actually used in the deleteAttrInNotifyCer() function

Reply:
> It seems useDefaultType, actionType and overrideMetadata parameter are not actually used in the deleteAttrInNotifyCer() function

I have removed useDefaultType, overrideMetadata from deleteAttrInNotifyCer() function in my latest commit  https://github.com/telefonicaid/fiware-orion/pull/4186/commits/c5392556585a013d6aea61c8df8cec0cb1779ba4 

I have used actionType parameter in deleteAttrInNotifyCer to set delete actionType.  Please suggest if any other changes are required.